validator/validator.go

```diff
@@ -0,0 +1,76 @@
+package validator
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"gopkg.in/Clever/kayvee-go.v2/logger"
+)
+
+var (
+	requiredKayveeFields = []string{
+		"source",
+		"title",
+		"level",
+	}
+
+	validLevels = map[string]bool{
```

Comment:
Oh okay, saw this used later on. The slice feels a little more intuitive to me, but this is fine. :)